app/templates/ui/app/detail/detail.html

```diff
@@ -34,3 +33,10 @@
     </div>
   </div>
 </div>
+<div class="row">
+  <div class="col-sm-12">
+    <button class="btn btn-primary" ui-sref="root.search">Cancel</button>
+    <button class="btn btn-primary" ui-sref="root.edit({uri: ctrl.uri})">Edit</button>
+    <button class="btn btn-primary" ng-click="ctrl.delete()">Delete</button>
```

Comment:
ctrl.delete() could show an 'are you sure' popup/popover if clicked. You can use angular-bootstrap for that..